### PR TITLE
docs: Adding WSL2 setup steps

### PR DESCRIPTION
**Pull request by PatrickLang**
KIND works in WSL2, so this quick doc shows how to set up the prerequisites. After that's done, the rest of the quick start applies as usual.

### CODE REVIEW

File: site/content/docs/user/using-wsl2.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    Enable-WindowsOptionalFeature -Online -FeatureName VirtualMachinePlatform, Microsoft-Windows-Subsystem-Linux
+    ```
+1. Reboot when prompted. 
+1. After the reboot, set WSL to default to WSL2. Open an admin PowerShell window and run `wsl --set-default version 2`.
+1. Now, you can install your Linux distro of choice by searching the Windows Store. If you don't want to use the Windows Store, then follow the steps in the WSL docs for [manual install](https://docs.microsoft.com/en-us/windows/wsl/install-manual).
+1. Start up your distro with the shortcut added to the start menu
+1. Install Docker - here's links for [Debian](https://docs.docker.com/install/linux/docker-ce/debian/), [Fedora](https://docs.docker.com/install/linux/docker-ce/fedora/), and [Ubuntu](https://docs.docker.com/install/linux/docker-ce/ubuntu/)
+1. Start the Docker daemon using init (not systemd) `sudo service docker start`. This needs to be done each time you start WSL2.
+
+Now, move on to the [Quick Start](/docs/user/quick-start) to set up your cluster with KIND.
```
Comment (aojea) [inline site/content/docs/user/using-wsl2.md]: is this link working? it's broken to me

**Comment by aojea** [comment]
/ok-to-test
Thanks

File: site/content/docs/user/using-wsl2.md (hunk 1 above)
Comment (PatrickLang) [reply site/content/docs/user/using-wsl2.md]: pinged on slack - it's working

File: site/content/docs/user/using-wsl2.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+title: "Using WSL2"
+menu:
+  main:
+    parent: "user"
+    identifier: "using-wsl2"
+    weight: 1
+---
+# Using WSL2
+
+KIND can run using Windows Subsystem for Linux 2 (WSL2) on Windows 10 Insider builds. All the tools needed to build or run KIND work in WSL2, but some extra steps are needed to switch to WSL2. This page covers these steps in brief but also links to the official documentation if you would like more details.
```
Comment (aojea) [inline site/content/docs/user/using-wsl2.md]: I remember some discussion about how to write KIND, cc: @BenTheElder

File: site/content/docs/user/using-wsl2.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+## Getting Windows 10 Insider Preview
+
+Download latest ISO at https://www.microsoft.com/en-us/software-download/windowsinsiderpreviewadvanced . Choose "Windows 10 Insider Preview (FAST) - Build 18912". If there's a later build number, get that instead and then you can skip the work to upgrade to 18917.
+
+### VM Settings
```
Comment (aojea) [inline site/content/docs/user/using-wsl2.md]: VM settings explain how to install Windows 10 Insider Preview in a VM? maybe `Installing on a virtual machine` is more explicit

File: site/content/docs/user/using-wsl2.md (hunk 2 above)
Comment (BenTheElder) [reply site/content/docs/user/using-wsl2.md]: I've given up on this :)

File: site/content/docs/user/using-wsl2.md (hunk 3 above)
Comment (PatrickLang) [reply site/content/docs/user/using-wsl2.md]: thanks! updated

File: site/content/docs/user/using-wsl2.md (hunk 2 above)
Comment (PatrickLang) [reply site/content/docs/user/using-wsl2.md]: The rest of the docs are lowercase "kind" so I updated to match

**Pull request merged**